lbry/lbry/schema/claim.py

```diff
@@ -214,6 +217,12 @@ def update(self, file_path=None, height=None, width=None, duration=None, **kwarg
                 kwargs.pop('fee_amount', None)
             )
 
+        source_stream_type = None
+        if self.source.name:
+            _, source_stream_type = guess_media_type(self.source.name)
```

Comment:
`guess_media_type(self.source.name)` gets called again on line 237. can this be cleaned up to avoid duplication?